app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/repository/SessionRemoteRepository.java

```diff
@@ -48,13 +48,15 @@ public Session apply(SessionRemote sessionRemote) throws Exception {
     public Single<List<Session>> listSessions() {
         return mService.getSessions().map(sessionRemotes -> {
                 List<Session> sessions = new ArrayList<>();
+
+                // TODO: Issue #143 - Move sorting to the presentation layer
+                Collections.sort(sessionRemotes, new SessionRemoteComparator());
```

Comment:
I agree that this operation can be costly. Not sure, however, how to solve this right now. It could be sorted by date on the server's side and provide more options to the user (like sort by type of session). I've opened issue #143 to move logic to the presenter, let's discuss it there.